### DISCUSSION TITLE
Cycler instructions standard

### DISCUSSION THREAD

**Discussion by TomTranter** [pybamm-team/PyBaMM#3268] [Ideas]
There are several concurrent efforts to standardize or at least improve communication between cyclers and simulations and data tools.
There is this thread by amplabs which some PyBaMM teams members have been involved with https://github.com/amplabs-ai/amplabs/discussions/33#discussioncomment-5736062
Battgenie, Galvanalyzer and probably others including Byterat and VoltaiQ probably also have similar concepts although maybe not be open source.
Worth having a discussion about state of progress and make sure we're not duplicating efforts

**Comment by martinjrobins** [comment]
Some other efforts in this direction:
- Galv is moving towards using JSON-LD to define metadata schemas, https://github.com/Battery-Intelligence-Lab/galv/issues/95
- This will probably follow [IntelLiGent-Data-Standard](https://github.com/Battery-Intelligence-Lab/IntelLiGent-Data-Standard) (@BradyPlanden)
- Galv and IntelLiGent-Data-Standard are using JSON-LD and the ontology EMMO (https://github.com/jsimonclark/EMMO) being developed by Simon Clark
- See also https://battery-modeling.slack.com/archives/C010BSCVB63/p1691507622023629 for related issues related to proprietry cycler data formats
